### PR TITLE
Location Handler Done

### PR DESCRIPTION
**Pull request by CeylinBrooks**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -33,6 +33,24 @@ function Location (locationData, cityDescrip){
...
 }
 
+app.get('/restaurants', restaurantHandler);
+
+function restaurantHandler(req,res){
+  const restaurantJSON = require('.data/restaturants.json');
+
+  const output = [];
+  for (let i = 0; i < restaurantJSON.nearby_restaurants.length; i++){
+    output.push(new Restaurant(restaurantJSON.nearby_restaurants[i].restaurant))
```
Comment (ncarignan) [inline server.js:43]: I like how you iterated over the json's properties all at once in one line

**Review by ncarignan — COMMENTED** [review]
So cool!

File: server.js (hunk 1 above)
Comment (nacerillo) [reply server.js:43]: Did you include the restaurant code just for the demo, or is this also part of the lab?

Comment (nacerillo) [reply server.js:43]: Can create weather without needed to use "req" as an argument.

**Pull request merged**